### PR TITLE
[Casting Text Visibility] add new tweak

### PR DESCRIPTION
**Pull request by img02**
Allows the user to change target + focus target casting text colour and font size. (I have bad eyesight)

image
image

would have liked to add a thicker edge or background but I couldn't get it to work :(

I based it off of TargetHp, so sorry if something's off.

### CODE REVIEW

**Comment by MidoriKami** [comment]
It seems to based this off of some of my other castbar tweaks, I strongly encourage you take a look at the recently rewritten versions of those tweaks and re-model this tweak off the new versions.

There's quite a lot of new SimpleTweaks framework that both makes the code simpler, but also makes it automatically enable/disable appropriately.

**Comment by img02** [comment]
Ok I'll take a look and rework things, thanks.

**Comment by img02** [comment]
I've redone things following TargetCastbarCountdown as closely as I could.

**Comment by MidoriKami** [comment]
Fantastic! Looks great! Thank you for taking the time to spruce it up.

File: Tweaks/UiAdjustment/CastingTextVisibility.cs [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+            }
+        }
+
+        protected override void Disable()
+        {
+            ResetTextNodes();
+            FreeAllNodes();            
+        }
+
+        [AddonPostUpdateAttribute("_TargetInfo", "_FocusTargetInfo")]
```
Comment (MidoriKami) [inline Tweaks/UiAdjustment/CastingTextVisibility.cs:96]: Did you mean for this to be `AddonPostUpdate` or `AddonPostRequestedUpdate`?
PostUpdate runs every frame, PostRequestedUpdate runs after the game has changed the element.

Comment (img02) [reply Tweaks/UiAdjustment/CastingTextVisibility.cs:96]: Nah I didn't, my bad.

**Pull request merged**